### PR TITLE
doc: add todo doc section with project setup tasks

### PR DESCRIPTION
**Pull request by aleksandrovB**
Added a new TODO.md file which contains a list of tasks
which need to be execute before the common work on the project.

### CODE REVIEW

File: TODO.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+1. Setup a new project
+2. Add default package structure in the project
+3. Integrate default code style
```
Comment (KalinGv) [inline TODO.txt:3]: Do we have to add style initially ?

Comment (aleksandrovB) [reply TODO.txt:3]: Yes i think we will need it.

Comment (KalinGv) [reply TODO.txt:3]: ok

**Review by KalinGv — APPROVED** [review]
LGTM !

**Pull request merged**